src/Apps/Ingress/Components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useCallback } from 'react';
+import {
+    IHomePageProps,
+    IHomePageStyleProps,
+    IHomePageStyles
+} from './HomePage.types';
+import { getStyles } from './HomePage.styles';
+import {
+    BaseButton,
+    classNamesFunction,
+    Pivot,
+    PivotItem,
+    Stack,
+    styled
+} from '@fluentui/react';
+import { useExtendedTheme } from '../../../../Models/Hooks/useExtendedTheme';
+import {
+    INGRESS_TRANSLATION_COMMON_PATH,
+    INGRESS_TRANSLATION_PATH
+} from '../../Models/Constants';
+import { useNavigationContext } from '../../Models/Context/NavigationContext';
+import {
+    NavigationContextActionType,
+    PageNames
+} from '../../Models/Context/NavigationContext.types';
+import { useTranslation } from 'react-i18next';
+
+const getClassNames = classNamesFunction<
+    IHomePageStyleProps,
+    IHomePageStyles
+>();
+
+const TRANSLATE_KEY_ROOT = `${INGRESS_TRANSLATION_PATH}.IngressHomePage`;
+const TRANSLATE_KEYS = {
+    // Common translation values
+    templates: `${INGRESS_TRANSLATION_COMMON_PATH}.templates`,
```

Comment:
translations does not show up
```suggestion
    templates: `${TRANSLATE_KEY_ROOT}.templates`,
```